numpy-stubs/__init__.pyi

```diff
@@ -368,6 +370,35 @@ class ndarray(_ArrayOrScalarCommon, Iterable, Sized, Container):
     @strides.setter
     def strides(self, value: _ShapeLike): ...
 
+    # Array conversion
+    def item(self, *args: int) -> Any: ...
+    def tolist(self) -> List[Any]: ...
+    @overload
+    def itemset(self, __value: Any): ...
+    @overload
+    def itemset(self, __item: _ShapeLike, __value: Any): ...
+    def tostring(self, order: str=...) -> bytes: ...
+    def tobytes(self, order: str=...) -> bytes: ...
+    def tofile(self, fid: Union[IO[Any], str], sep: str, format: str): ...
+    def dump(self, file: str): ...
+    def dumps(self) -> bytes: ...
+    def astype(self,
+               dtype: _DtypeLike,
+               order: str=...,
+               casting: str=...,
+               subok: bool=...,
+               copy: bool=...) -> ndarray: ...
+    def byteswap(self, inplace: bool=...) -> ndarray: ...
+    def copy(self, order: str=...) -> ndarray: ...
+    def view(self,
+             dtype: Union[_DtypeLike, Type]=...,
+             type: Any=...) -> ndarray: ...
```

Comment:
This should probably substitute `Type` -> `Type[ndarray]` and `Any` -> `Type[ndarray]`.

Reply:
Thank you! Fixed. Covered methods with tests